### ISSUE TITLE
nested grouped bars

### ISSUE THREAD

**Issue by iwan-uschka** [FormidableLabs/victory#1632]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

I would like to create a chart showing daily data grouped by weeks. I was able to do so for regular bars:

<img width="305" alt="Bildschirmfoto 2020-07-02 um 00 06 49" src="https://user-images.githubusercontent.com/2256579/86295949-f631da80-bbf7-11ea-8b69-94f3d6070360.png">

I was able to do so for stacked bars:

<img width="303" alt="Bildschirmfoto 2020-07-02 um 00 06 53" src="https://user-images.githubusercontent.com/2256579/86295972-ff22ac00-bbf7-11ea-96c2-1b6652b2a38a.png">

"KW" means "Kalenderwoche" in German and "calendar week" in English. As you can see each calendar week shows 7 days with a group offset to create some space between the calendar weeks.

I wasn't able to do so for grouped bars. I would like to have 2 (or mutliple) bars per day but not on top of each other, like done for stacked bars but next to each other.

Is this possible?

**Comment by joaovitorqueiroz** [comment]
@iwan-uschka you can apply the offset as in the example: [https://formidable.com/open-source/victory/docs/victory-group](url)

**Comment by iwan-uschka** [comment]
Uff, long time ago :) I had to think a little what i was trying to achieve which means i haven't asked the question clearly enough in the first place. 

I used `VictoryGroup` for grouping days of a week and having different margins for days within in a week and between the weeks themselves (shown in [thread opening](https://github.com/FormidableLabs/victory/issues/1632#issue-649355930)).
I thought it would be cool to have the option of creating nested groups (like shown in the picture) to achieve something like this:

![victory-nested-groups](https://user-images.githubusercontent.com/2256579/98340411-9caa4400-200d-11eb-9197-6812fbca85d3.png)

Grouping days as a week. Grouping different data as/in/for a day. The weeks have a certain margin between each other. Days within a week have a different margin and bars of a day have again a different margin. Sounds like nested groups 🤯

This is not an urgent issue anymore but if anyone has found a solution to this, I would be happy to hear about it. 🙃
